package.json

```diff
@@ -283,7 +284,6 @@
     "angular-route": "1.8.3",
     "angular-sanitize": "1.8.3",
     "ansicolor": "1.1.100",
-    "app": "link:./public/app",
```

Comment:
How do these resolve now? I was expecting to find them in tsconfig `paths` but they're not there? 🤔

Reply:
So, webpack is able to resolve them thanks to

```js
modules: ['node_modules', path.resolve('public')],
```

Typescript resolves them thanks to `"baseUrl": "public/",` in the tsconfig.

Reply:
Ah ok, thanks for the explanation. 👍